### PR TITLE
chore: add Sentry to project

### PR DESCRIPTION
**Pull request by viniciuslacerda0**
## Description

Add Sentry to project
Config Routes and loggers

## Related Issues

Use keywords like 'close' or 'solves' to link this PR to an issue.
For example:

Closes #12345
Unblocks #54321

## Progress

Use checkboxes to show that you did and what you have to do. For example:

- [x] Add Sentry to project;
- [x] Config Routes and loggers;

## How to test it

1 - yarn config-env
2 - yarn
3 - yarn dev
4 - Create an error
5 - Access Sentry to verify that error log is there 

## Visual reference

image

image

## Change logs

Add Sentry to project;
Config Routes and loggers;

### CODE REVIEW

File: src/services/logging.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    }
+    addBreadcrumb({
+      category: 'graphQL error',
+      data: { error, extras },
+      level: 'error',
+      message: `GraphQL error`,
+    });
+    captureMessage(`GraphQL error`);
+  });
+};
```
Comment (gustavolbs) [inline src/services/logging.ts:98]: **question:** Does Sentry works with GraphQL or are we going to use GraphQL?

**Review by gustavolbs — APPROVED** [review]

**Comment by gustavolbs** [comment]
Please, add the related issue at PR description

**Comment by gustavolbs** [comment]
**Question:** Does this change add value to our product? I mean, we have a lot of things we have to do and we're still setting up Sentry or ConventionalCommits while we have delayed tasks that are more valuable for our deliverables.

File: src/services/logging.ts (hunk 1 above)
Comment (viniciuslacerda0) [reply src/services/logging.ts:98]: I added it by mistake, i’ll change that

**Comment by viniciuslacerda0** [comment]
> **Question:** Does this change add value to our product? I mean, we have a lot of things we have to do and we're still setting up Sentry or ConventionalCommits while we have delayed tasks that are more valuable for our deliverables.

It adds because the sentry can be used like an analytics! About the commit lint it is a fix since I’ve forgot some key things when i was creating the project base

**Comment by gustavolbs** [comment]
> > **Question:** Does this change add value to our product? I mean, we have a lot of things we have to do and we're still setting up Sentry or ConventionalCommits while we have delayed tasks that are more valuable for our deliverables.
> 
> It adds because the sentry can be used like an analytics! About the commit lint it is a fix since I’ve forgot some key things when i was creating the project base

I don't agree with you on this. We don't even have the login page, let alone something to be analyzed

**Comment by viniciuslacerda0** [comment]
> > > **Question:** Does this change add value to our product? I mean, we have a lot of things we have to do and we're still setting up Sentry or ConventionalCommits while we have delayed tasks that are more valuable for our deliverables.
> > 
> > 
> > It adds because the sentry can be used like an analytics! About the commit lint it is a fix since I’ve forgot some key things when i was creating the project base
> 
> I don't agree with you on this. We don't even have the login page, let alone something to be analyzed, but if you think it adds, no problem. I don't choose what will be added to the sprint haha.

I think we should discuss on the meeting tomorrow our priorites and power up some taskes, no prob!

**Pull request merged**